README.md

```diff
@@ -24,15 +24,15 @@ c = a;
 ### Your solution here:
 1.  What is `a`?
 ```
-a is ...
+a is bongos
 ```
 2.  What is `b`?
 ```
-b is ...
+b is true
 ```
 3.  What is `c`?
 ```
-c is ...
+c is 1
```

Comment:
c is 'bongos'

a was changed re-assigned to bongos
then c was re-assigned as a after it was given the value of bongos